src/videojs-contrib-hls.js

```diff
@@ -177,7 +178,10 @@ export default class HlsHandler extends Component {
         /* eslint-enable dot-notation */
       }
       this.masterPlaylistController_.useAudio();
```

Comment:
This line must be moved to `loadedmetadata` handler for it to have any effect. Right now we don't select an audio track if the source is demuxed (pretty much everything but Advanced BipBop.)